patternfly/patternfly-elements
# fix: pfe-tabs set default index to 0 if null provided

## pfe-tabs

If null is set for `selectedIndex`, the tabset should always use 0.

### Preview

- [Link](https://deploy-preview-1275--happy-galileo-ea79c4.netlify.app/elements/pfe-tabs/demo/) 

### Testing instructions

Be sure to include detailed instructions on how your update can be tested by another developer.

1. Open the [demo page](https://localhost:8000/elements/pfe-tabs/demo/) locally using `npm run dev pfe-tabs`
2. In the mark-up for the demo file (`/elements/pfe-tabs/demo/index.html`) add a `selected-index="null"` to the first `pfe-tabs` element on the page.
3. Inspect after refresh and confirm that `null` was changed to `0` on upgrade.

#### Regression testing
Validate no regressions to `pfe-content-set`.


#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
